src/sensors.c

```diff
@@ -155,7 +160,9 @@ void init_sensors(void)
   mpu6050_register_interrupt_cb(&imu_ISR);
 
   uint16_t acc1G;
-  mpu6050_init(true, &acc1G, &gyro_scale, _params.values[PARAM_BOARD_REVISION]);
+  float gyro_scale_to_urad;
+  mpu6050_init(true, &acc1G, &gyro_scale_to_urad, _params.values[PARAM_BOARD_REVISION]);
+  gyro_scale = 1000000.0*gyro_scale_to_urad;
```

Comment:
Okay, ignore that last comment. Just looking at the IMU data streaming over mavlink, it looks the units are at least close to correct as it is right now. Which means that it's probably something goofy with this line: (`drv_mpu6050.c:220`)

    // 16.4 dps/lsb scalefactor for all Invensense devices
    *gyroScale = (4.0f / 16.4f) * (M_PI / 180.0f) * 0.000001f;

I think _he's_ got his units messed up. This also goes back to your observation in https://github.com/BYU-MAGICC/ROSflight2/issues/29#issuecomment-230694862. If you get rid of the `0.000001f`, then it'd be close to actually getting you into rad, although the `4.0f` is still a mystery. But with the `0.000001f`, it puts you in units of Mrad/s? Something goofy is definitely going on there.